Dockerfile.riscv64

```diff
@@ -1,73 +1,20 @@
 #==============================
-# Binaries Builder
+# Builder Stage
```

Comment:
Please merge this patch in the commit where you introduced `Dockerfile.riscv64`. We try to avoid as much as possible modifying code that we introduced in some previous commit of the same PR.

Reply:
Resolved by squashing.